api/v1alpha1/remoteWrite_types.go

```diff
@@ -32,6 +34,15 @@ type RemoteWriteSecretSpec struct {
 	Data map[string][]byte `json:"data,omitempty"`
 }
 
+type RemoteWriteStatus struct {
+	ConfiguredPrometheuses []RemoteWriteStatusConfiguredPrometheus `json:"configuredPrometheuses"`
```

Comment:
it's very long name for type `RemoteWriteStatusConfiguredPrometheus`

I would prefer using an upstream type instead,`ObjectReference`
https://pkg.go.dev/k8s.io/api/core/v1#ObjectReference

in fact, you can choose what you want to supply in this reference, which means the name and namespace is enough in this case.

Reply:
Added this, good point. Thanks